app/app.go

```diff
@@ -151,10 +151,9 @@ const (
 	// NOTE: In the SDK, the default value is 255.
 	AddrLen = 20
 
-	FlagMemIAVL  = "store.memiavl"
-	FlagAsyncWAL = "store.memiavl-async-wal"
```

Comment:
do we still need FlagAsyncWAL?

Reply:
yes, it has to be like this in app setup code, I moved to separate module to look better